### PR TITLE
Show post on subscribe

### LINKED ISSUE #122: Add post during subscribing to the feed
Now, when you subscribe to the feed you will not see the new feed in Slow/Fast section (because there are no posts after subscription to the feed).

Let’s add the latest post from the new feed.

- [ ] Change `addPreviewCandidate()` in `core/preview.ts` by adding post adding code from [`core/refresh`](https://github.com/hplush/slowreader/blob/main/core/refresh.ts#L109-L124).
- [ ] Find a way to re-use post adding code between `core/preview.ts` and `core/refresh`. For instance, by creating some helper in `core/post.ts`.
- [ ] Add tests

### PR DESCRIPTION
**Pull request by chafilin**
Fixes #122

Added last post after subscribe on feed. 

# Motivation

# Checklist

- [X] Don’t rush. Check all changes in PR again.
- [X] Run `pnpm test`.
- [X] Think about changing documentation.
  - If you added a script to `scripts/`, add a comment with a description.
  - If you added a new folder, add its description to the project’s `README.md`.
  - If you added config, describe how we use this tool in the config’s comment.
  - If you added something to the project’s architecture, describe it in the project’s `README.md`.
  - Try to focus on “why?”, not “how?”.
- [X] If you added a new dependency, check our [requirements](../README.md#dependencies).
- [X] Think about testing
  - If you added a feature, add unit tests.
  - If you added a new state to the UI, add visual tests.
  - If you fixed the bug, think about preventing bug regression in the future.
- If you changed web client:
  - [x] Think about moving code to `core/`. What code will also be useful on other platforms?
  - [X] Run `pnpm size` and check the difference in the JS bundle size. Is it relevant to the changes? Change the limit in `web/.size-limit.json` if necessary.
  - [X] The UI was checked in Chrome and Firefox (and Safari or Epiphany if you have them).
  - [X] Think about keyboard UX. Is it easy to use the new feature with only one hand on a keyboard? Is it easy to understand what keys to press?
  - [x] Think about HTML semantics.
  - [X] Think about accessibility. Check a11y recommendations. Think about how screen reader users will use the tool. Is it easy to use on a screen with bad contrast?
  - [X] Think about translations.
  - [X] Think about right-to-left languages. What parts of the screen should be mirrored for Arabic or Hebrew languages?
- If you changed the colors token in the web client:
  - [X] Think about app loading styles inlined in `index.html`.
- If you changed `core/`:
  - [X] Think about making types more precise. Can you better explain data relations by type?
  - [X] Think about conflict resolution. We don’t need some very smart changing merging; just 2 changes of the same item on different clients should not break the database. What if the user changes an item on one machine and removes it on another?
  - [X] Think about log and storage migration.
- If you changed English translations:
  - [X] Change translation ID if you change the meaning of the text.

### CODE REVIEW

File: web/.size-limit.json [hunk 1]
```diff
@@ -8,6 +8,6 @@
     "name": "All scripts to execute",
     "path": "dist/assets/*.js",
     "brotli": false,
-    "limit": "210 KB"
+    "limit": "210.04 KB"
```
Comment (ai) [inline web/.size-limit.json:11]: Increase it with 5 KB step

**Comment by ai** [comment]
Why Draft PR?

**Comment by chafilin** [comment]
Created draft because have two comments: 
1. I am not sure that helper is needed, because addPost is enough for this case. 
2. I see UX issue when we subscribe to feed and then change filter to slow, but last post already have added to fast

**Comment by ai** [comment]
Thanks! Congrats with the first PR!

**Comment by ai** [comment]
Also, maybe you want to take similar task? https://github.com/hplush/slowreader/issues/148

There we need to change old post’s reading type based on new default feed type or filter changes (re-run filter logic for old posts).

**Comment by chafilin** [comment]
> Thanks! Congrats with the first PR!

Thanks!

> Also, maybe you want to take similar task? #148
> 
> There we need to change old post’s reading type based on new default feed type or filter changes (re-run filter logic for old posts).

Sure, I take it

**Pull request merged**